insights/client/connection.py

```diff
@@ -170,13 +170,28 @@ def _http_request(self, url, method, log_response_text=True, **kwargs):
         Returns
             HTTP response object
         '''
-        logger.log(NETWORK, "%s %s", method, url)
+        log_message = "{method} {url}".format(method=method, url=url)
+        if "data" in kwargs.keys():
+            log_message += " data={data}".format(data=kwargs["data"])
+        if "json" in kwargs.keys():
+            log_message += " json={json}".format(json=json.dumps(kwargs["json"]))
+        if "headers" in kwargs.keys():
+            log_message += " headers={headers}".format(headers=kwargs["headers"])
+        if "files" in kwargs.keys():
+            attachments = []
+            for name, content in six.iteritems(kwargs["files"]):
+                if isinstance(content, tuple):
+                    attachments.append("{name}:{file}".format(name=name, file=content[0]))
```

Comment:
I'm not sure about logging the content of the files:
- they might be binaries
- they might contain sensitive information

let's keep only the filenames for now

Reply:
The `content[0]` is the filename; files in a request are defined as
```py
[("field name", ("filename", file_pointer, "content type"))]
# or the dict equivalent
```